src/python/pants/rules/core/test.py

```diff
@@ -25,6 +26,12 @@ class TestOptions(GoalSubsystem):
   """Runs tests."""
   name = "test"
 
+  @classmethod
+  def register_options(cls, register) -> None:
+    super().register_options(register)
+    register('--debug', type=bool, fingerprint=True, default=False,
```

Comment:
You can drop `fingerprint=True`. `fingerprint` is irrelevant in a V2 world; we only need it when the subsystem may be used by V1 code.